src/Query/Mysql/Acos.php

```diff
@@ -1,22 +1,26 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Let's disable this rule for now. Enabling strict types everywhere can be a breaking change.